extensions/adobe/experience/decisioning/decisionevent.schema.json

```diff
@@ -0,0 +1,116 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/decisionevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "DecisionEvent",
+  "type": "object",
+  "description": "A DecisionEvent is used to capture observations about the outcome and context of a decision activity. The DecisionEvent captures information about how the decision made, when it was occurring, what options were proposed (chosen) and what contextual state existed that either influenced the decision or could be observed during the decision process. The decision event also captures the proposition ID, a globally unique identifier that can be used to correlate the decision to other events. DecisionEvents are not only relatable to Experience Events that impacted a decision but also to ExperienceEvents that are a direct response to a proposition. It is the expectation that applications reference the proposition ID in every experience event that was influended by the propositions. A user profile's proposition-response history is maintained with these proposition IDs.",
+  "meta:abstract": false,
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/time-series",
+
+    "https://ns.adobe.com/experience/decisioning/proposition#/definitions/propositions-mixin",
+
+    "https://ns.adobe.com/xdm/context/identitymap",
+    "https://ns.adobe.com/xdm/context/experienceevent-advertising",
```

Comment:
Hi @jbeckert, this schema appears to mixin extensions intended for ExperienceEvents, but is not itself an experience event.

Reply:
Correct, let me know if this is a problem. There is a new schema for decision propositions. A proposition is made when a decision request is issued inside the platform. That fact is recorded by a DecisionEvent. As per our discussions last year we don't consider this an ExperienceEvent just yet. On the other hand there is an event and that gets recorded in the service as well. Both the DecisionEvent and the ExperienceEvent carry through the proposition data as a mixin property. When an ExperienceEvent echoes back the propositionID and even records a response (clicked, accepted, etc) then the service can correlate this back to the decision event. This is what we are calling the proposition-response history. It it important that we can distinguish between proposed offers that were responded to vs those that were not responded to (yet).

Reply:
I solved the problem by creating a relationship between the mixin "https://ns.adobe.com/experience/campaign-experienceevent" (in the "experience" namespace) and "https://ns.adobe.com/experience/decisioning/decisionevent" (from an "experience" sub-namespace) instead of directly mixing in the extensions from the "xdm/context" namespace

Reply:
I solved the problem by creating a relationship between the existing mixin "https://ns.adobe.com/experience/campaign-experienceevent" and "https://ns.adobe.com/experience/decisioning/decisionevent" instead of directly mixing in the extensions meant for core experience event class. At least the class decisioningevent and the mixin campaign-experienceevent are in closely related namespaces